src/androidTest/java/de/dennisguse/opentracks/content/data/TestDataUtil.java

```diff
@@ -73,4 +79,26 @@ public static void insertTrackWithLocations(ContentProviderUtils contentProvider
         contentProviderUtils.insertTrack(track);
         contentProviderUtils.bulkInsertTrackPoint(trackPoints, track.getId());
     }
+
+    /**
+     * Creates a Waypoint with a photo.
+     *
+     * @param context  The context.
+     * @param trackId  The track id.
+     * @param location The location.
+     * @return the Waypoint created.
+     */
+    public static  Waypoint createWaypointWithPhoto(Context context, long trackId, Location location) {
+        String photoUrl = "";
+        try {
+            File dstFile = new File(FileUtils.getImageUrl(context, trackId));
+            dstFile.createNewFile();
+            Uri photoUri = FileUtils.getUriForFile(context, dstFile);
+            photoUrl = photoUri.toString();
+        } catch (IOException ioe) {
```

Comment:
Actually, you can just throw the IOException directly as this is only test code :)

Reply:
Ok. Done :)